cou/cli.py

```diff
@@ -145,4 +145,4 @@ async def entrypoint() -> int:
         return 0
     except Exception as exc:  # pylint: disable=broad-exception-caught
         logging.exception(exc)
-        return 1
+        sys.exit(1)
```

Comment:
There is still return 0 and I think there should be no return.

Reply:
ok